bindata/network/network-metrics/ipsec-rules.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
i think move to manifests file and name `07-prometheus-rules.yaml`. WDYT?

This dir is for network metrics daemon.

Reply:
The file was not applied during the installation in the manifests directory, I am not sure why. On the other hand, this is kind of network metric...